CRAMv3.tex

```diff
@@ -732,30 +736,67 @@ \subsection{\textbf{Slice header block}}
 
 The slice header block is never compressed (block method=raw). For reference mapped 
 reads the slice header also defines the reference sequence context of the data 
-blocks associated with the slice. Mapped and unmapped reads can be stored within 
-the same slice similarly to BAM file. Slices with unsorted reads must not contain 
-any other types of reads.
+blocks associated with the slice. Mapped reads can be stored along with
+\textbf{placed unmapped}\footnote{Unmapped reads can be \textit{placed} or \textit{unplaced}.
+By placed unmapped read we mean a read that is unmapped according to 
+its BAM bit flags but has position fields filled in, thus "placing"  it on a reference sequence. In contrast, 
+unplaced unmapped reads have have a reference sequence ID of -1 and alignment position of 0.}
+reads on the same reference within the same slice, as is possible in BAM.  
+
+Slices with the Multiple Reference flag (-2) set as the sequence ID in the header may contain reads
+mapped to multiple external references, including unmapped\footnotemark[\value{footnote}] reads (placed on these references or unplaced), 
+but multiple embedded references cannot be combined in this way.  When multiple references are 
+used, the RI data series will be used to determine the reference sequence ID for each record. This 
+data series is not required when only a single reference is used within a slice.
```

Comment:
Pedant mode: Maybe "not present" to be explicit that it is banned.

This doesn't actually matter in modern implementations as we put each data series into its own block, but the earlier crams used HUFFMAN, BETA, GAMMA etc and interleaved everything into the CORE block.  In that situation it is important to *not* write out the RI field when it's not needed otherwise we get out of sync when decoding.

Reply:
done